public-images/public-images-reference/artifacts.rst

```diff
@@ -0,0 +1,1495 @@
+
+Root tarball
+~~~~~~~~~~~~
+.. list-table::
+   :widths: 1 2
+   :header-rows: 0
+
+   * - Extension
+     - ``<artifact>-root.tar.xz``
+   * - Example filename
+     - ``noble-server-cloudimg-amd64-root.tar.xz``
+   * - Format description
+     - Tar archive compressed with XZ (LZMA2) containing a root file system for various architectures.
+   * - Use cases
+     - These files are used for deploying base system images in virtual machines and containers. You can use ``.root.tar.xz`` files to distribute pre-configured root file systems that can be deployed directly into virtual machines or container runtimes like Docker or Kubernetes.
+
+SquashFS
+~~~~~~~~
+.. list-table::
+   :widths: 1 2
+   :header-rows: 0
+
+   * - Extension
+     - ``.squashfs``
+   * - Example filename
+     - ``noble-server-cloudimg-amd64.squashfs``
+   * - Format description
+     - SquashFS is a compressed read-only file system format.
+   * - Use cases
+     - SquashFS files are used for embedding file systems in read-only environments, often in embedded systems or live CDs. In cloud environments, they are used for distributing lightweight operating system images that are ready to use. You can use a ``.squashfs`` file containing a minimal Linux distribution to create container images that boot quickly and require minimal storage space.
+
+Tarball (gzip)
+~~~~~~~~~~~~~~
+.. list-table::
+   :widths: 1 2
+   :header-rows: 0
+
+   * - Extension
+     - ``<artifact>.tar.gz``
+   * - Example filename
+     - ``noble-server-cloudimg-amd64.tar.gz``
+   * - Format description
+     - ``.tar.gz`` is an archive format, often called a tarball, that combines files into a single file and compresses it using ``gzip`` compression.
+   * - Use cases
+     - Our ``.tar.gz`` archives are used to distribute complete file system images along with the kernel for various operating systems and virtualisation platforms. They allow extraction and booting of the entire system on compatible hardware or virtual machines.
+
+Vagrant Box
+~~~~~~~~~~~
+.. list-table::
+   :widths: 1 2
+   :header-rows: 0
+
+   * - Extension
+     - ``.box``
+   * - Example filename
+     - ``jammy-server-cloudimg-amd64-vagrant.box``
+   * - Format description
+     - The Vagrant Box format is used to package and distribute virtual machine environments managed by Vagrant.
+   * - Use cases
+     - These files contain a virtual machine image along with metadata required for Vagrant. Vagrant simplifies the creation and provisioning of virtual environments, making it easier to manage and share development environments across different systems. You can use a ``.box`` file along with a supported provider to quickly set up environments with specific configurations, tools and dependencies. All of Vagrant Boxes are provider specific, with our Boxes having been built for VirtualBox.
+
+Virtual Hard Disk (VHD)
```

Comment:
The VHDs published on `cloud-images.ubuntu.com` do not contain standard VM images but tailored images for Azure cloud. Those images would not work outside of Azure as cloud-init is only configured to listen to the AzureDatasource. Please make sure this is clearly explained here: those images are only for Azure not for on-prem hyper-Vs.

Reply:
Thank you for clarifying these Azure specific points. I've added on to the end of the VHD section to summarize the above.